cmd/end2endtest/plaintext.go

```diff
@@ -54,16 +54,24 @@ func (t *E2EPlaintextElection) Run() error {
 	// Send the votes (parallelized)
 	startTime := time.Now()
 
-	log.Infof("enqueuing %d votes", len(t.voterAccounts))
+	//log.Infof("enqueuing %d votes", len(t.voterAccounts))
```

Comment:
I'd personally delete a log line if it's no longer useful. It's always in the git history.